shell/browser/browser_linux.cc

```diff
@@ -25,11 +29,19 @@
 
 namespace electron {
 
+const char kFreedesktopPortalName[] = "org.freedesktop.portal.Desktop";
+const char kFreedesktopPortalPath[] = "/org/freedesktop/portal/desktop";
+const char kFreedesktopPortalBackground[] = "org.freedesktop.portal.Background";
+
 namespace {
 
 const char kXdgSettings[] = "xdg-settings";
 const char kXdgSettingsDefaultSchemeHandler[] = "default-url-scheme-handler";
 
+scoped_refptr<dbus::Bus> bus_;
```

Comment:
It doesn't look like these are used, and we probably wouldn't want to put this in the anonymous namespace!

Reply:
Removed!